spec/support/public/test.js

```diff
@@ -28,10 +28,10 @@ $(function() {
       $("#changes_on_keypress").text(increment)
     })
     .focus(function(event) {
-      $("#changes_on_focus").text("Focus")
+      $("#changes_on_focus").text(`Focus (${event.originalEvent.constructor.name})`)
```

Comment:
I would like tests to cover the change in this PR, so I added the event constructor name here. On `main`, this text would be `Focus (Event)`. On this branch, it's `Focus (FocusEvent)`.